sdk/security/keyvault/azcertificates/models.go

```diff
@@ -461,20 +433,8 @@ type ImportCertificateParameters struct {
 	Tags map[string]*string `json:"tags,omitempty"`
 }
 
-// IssuerAttributes - The attributes of an issuer managed by the Key Vault service.
```

Comment:
Commenting close: `ImportCertificateParmaeters should be `[]byte` for `Base64EncodedCertificate`, which you agreed to fix in APIView: https://apiview.dev/Assemblies/Review/66481607611444989da6998d76c414d6?diffRevisionId=b4b48fc855284881971c5cf94426954e&diffOnly=False&revisionId=6811d1794b714328a2da849ae047ade4&doc=False#4643f49b0d4c46be81f3ddd4b0e3d88d

Reply:
Investigated further and talked with Charles, it turns out changing the type to []bytes is not a good idea because it would require a nuanced convenience layer there that doesn't translate well to Go and doesn't provide enough value to users to justify the change.